src/style.css

```
@@ -0,0 +1,97 @@
+@font-face {
+  font-family: "Avenir-Book";
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url("./fonts/Avenir-Heavy.ttf");
+}
+
+* {
+  padding: 0;
+}
+
+body {
+  font-weight: 300;
+  font-family: "Avenir", sans-serif;
+}
+
+[data-qa="big"] {
+  width: 100%;
+  height: 70px;
+}
```

use `big` class here 
```suggestion
.big {
  width: 100%;
  height: 70px;
}
```